### DISCUSSION TITLE
Add custom vendor for Openjdk

### DISCUSSION THREAD

**Discussion by Hax4us** [termux/termux-packages#8492] [Ideas]
i noticed one thing that there is no any specific system property in current build of openjdk-17 by which i can differentiate Android (arm) and Linux (arm) so i was thinking that we should set some property like `--with-vendor-name` Android or Termux, what you think about it ?

and yes i know there are many things i can do from code but i think it would be elegant solution.
And i saw many projects on github which supports Android but failed to detect Android platform on Termux. bcz all of them were using system properties for platform detection like `java.vendor` , `java.runtime.name` , `java.vm.vendor` etc
So its clear those projects were made for Android but with dalvikvm. and dalvikvm has all properties set appropriately.

**Comment by thunder-coding** [comment]
I feel this is eligible for a bug report. Not a Java developer but there must be someone who can give proper guidance/views on what should be done for this.
